packages/stockflux-core/src/services/ChildWindowLauncher.js

```diff
@@ -1,18 +1,44 @@
 import * as OpenfinApiHelpers from '../openfin-api-utils/openfinApiHelpers';
 
+const windowAlreadyExist = async (windowName, childWindows) => {
+  for (let i = 0; i < childWindows.length; i++) {
```

Comment:
Maybe we should change this to `for...of` loop?

Reply:
Changed to `for...of `.